server/item/enchantment_rarity.go

```diff
@@ -0,0 +1,18 @@
+package item
+
+var (
+	// EnchantmentRarityCommon is the common enchantment rarity.
+	EnchantmentRarityCommon = EnchantmentRarity{Name: "Common", ApplyCost: 1}
+	// EnchantmentRarityUncommon is the uncommon enchantment rarity.
+	EnchantmentRarityUncommon = EnchantmentRarity{Name: "Uncommon", ApplyCost: 2}
+	// EnchantmentRarityRare is the rare enchantment rarity.
+	EnchantmentRarityRare = EnchantmentRarity{Name: "Rare", ApplyCost: 4}
+	// EnchantmentRarityVeryRare is the very rare enchantment rarity.
+	EnchantmentRarityVeryRare = EnchantmentRarity{Name: "Very Rare", ApplyCost: 8}
```

Comment:
Might be worth making `EnchantmentRarity` an interface and having unexported implementations for each rarity, then declaring the variables like `var EnchantmentRarityCommon enchantmentRarityCommon`. That way users cannot edit these variables.

Reply:
Changed.